include/gridtools/thread_pool/hpx.hpp

```diff
@@ -0,0 +1,31 @@
+/*
+ * GridTools
+ *
+ * Copyright (c) 2020, ETH Zurich
+ * All rights reserved.
+ *
+ * Please, refer to the LICENSE file in the root directory.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#pragma once
+
+#include <hpx/parallel/algorithm.hpp>
+#include <hpx/runtime_fwd.hpp>
```

Comment:
Let's do this instead for now. The header locations are somewhat unstable on master right now.

```suggestion
#include <hpx/include/parallel_for_loop.hpp>
#include <hpx/include/runtime.hpp>
```